Add/health sector

dlousa92 opened the pull request:
Create new sector and its appropriate subsectors & organize all posts into its sector and subsector.

leezaweb commented:
Thank you thank you thank you for the clear commit messages. Truly an inspiration and something for me to aspire to. 😁😂 Everything looks good, but I'm wondering if the dropdown on the homepage and on the header nav can be reduced from that 20vh height. If you ever see something like this and are curious yourself, you can add that to your PR comment. Were you asked to comment out the subsector list or remove that code entirely? Thank you for doing this. Please remove the package-lock  from your PR. That must be leftover from when you ran npm start.

dlousa92 commented:
I was also wondering the same thing! I'll make sure to post any future ideas/questions in a comment when I make pull requests. I went ahead and deleted the package lock file, oops. I commented out the code for testing purposes but now I went ahead and deleted it. Thank you for looking over this and the good feedback!
